documentation/planning.md

```diff
@@ -71,37 +86,38 @@ As explained above, TesLight supports up to **8 different zones**.
 How you select these zones is up to you, but keep in mind that you must be able to install and wire the LEDs in the zones.
 
 Since TesLight was designed with the Tesla Model 3 and Model Y in mind, there is a recommended configuration of the zones.
-This is the same configuration as shown in the following video (click to open).
+This is the same configuration as shown in the following video (click to open). Any TesLight light effects will also work best with this recommended zone configuration.
 
 [![TesLight Demo](https://img.youtube.com/vi/_N5h1IViB-E/0.jpg)](https://www.youtube.com/watch?v=_N5h1IViB-E)
 
-The following table assumes that the TesLight controller is placed below the center console for easy access and wiring. (It would be best if the antenna of the ESP32 points to the front and the USB port to the back of the vehicle. This way we ensure consistent acceleration directions from the sensor for all users.)
```

Comment:
Orientation of the TesLight controller is mandatory, you dropped it. --> Please re-add